setup.py

```diff
@@ -19,8 +19,8 @@
       py_modules=['Chandra.axTime3', 'Chandra.Time'],
       version='3.20',
       zip_safe=False,
-      test_suite="Chandra.test_Time",
-      packages=['Chandra'],
-      package_dir={'Chandra': 'Chandra'},
+      test_suite="Chandra.Time.test_Time",
+      packages=['Chandra', 'Chandra/Time'],
```

Comment:
Without the package_dir, it looked to me like setuptools was not bothering to make Chandra/Time and copy anything into it.